module/module_interceptors_test.go

```diff
@@ -32,9 +32,6 @@ import (
 func TestOpID(t *testing.T) {
 	ctx := context.Background()
 
-	if runtime.GOARCH == "arm" {
-		t.Skip("skipping on 32-bit ARM -- subprocess build warnings cause failure")
```

Comment:
not sure how much we care about arm32 these days but no objection to adding back